saul-examples/src/test/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/ModelsTest.scala

```diff
@@ -82,9 +82,9 @@ class ModelsTest extends FlatSpec with Matchers {
     results.perLabel.foreach {
       result =>
         result.label match {
-          case "A0" => result.f1 should be(0.95 +- 0.05)
-          case "A1" => result.f1 should be(0.95 +- 0.05)
-          case "A2" => result.f1 should be(0.8 +- 0.03)
+          case "A0" => result.f1 should be >= 0.9
+          case "A1" => result.f1 should be >= 0.9
+          case "A2" => result.f1 should be >= 0.8
```

Comment:
Sorry for being nitpicky. It bothers be when the tests are not upper-bounded. Could you make it more strict?